### DISCUSSION TITLE
Is there a way to hide memory usage of internal libraries?

### DISCUSSION THREAD

**Discussion by AvichalChadha** [bloomberg/memray#315] [Q&A] [answered]
Hi everyone, 
Is there a way to hide memory usage of the internal libraries and only get it for the functions present in the codebase

**Comment by pablogsal** [comment, marked as answer]
Thanks for reaching out @AvichalChadha !

> Is there a way to hide memory usage of the internal libraries and only get it for the functions present in the codebase

Not currently, but you can search in the flamegraph for the functions you are interested in (which I know it is not the same). You can also hide the imports using a checkbox with the latest version of `memray`, which may help simplify the flamegraph.

We will think about this use case and will try to think of something, as I think this can be quite useful.

**Reply by AvichalChadha** [reply]
Okay, thanks for clarifying that.
